### ISSUE TITLE
Test failures with Java 11

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-web#128]
```
org.eclipse.xtext.xbase.web.test.ContentAssistTest > testExtendsType FAILED
    org.junit.ComparisonFailure at ContentAssistTest.java:46

org.eclipse.xtext.xbase.web.test.ContentAssistTest > testPropertyType FAILED
    org.junit.ComparisonFailure at ContentAssistTest.java:46

```

**Comment by cdietrich** [comment]
no proposals are created

**Comment by cdietrich** [comment]
```
org.junit.ComparisonFailure: expected:<...ntries = ArrayList ([
    ContentAssistEntry [
      prefix = "AbstractQu"
      proposal = "AbstractQueue"
      description = "java.util.AbstractQueue"
      textReplacements = ArrayList (
        replace region [0 / length: 0] '
        import java.util.AbstractQueue'
      )
      editPositions = ArrayList ()
      kind = "REFERENCE"
    ],
    ContentAssistEntry [
      prefix = "AbstractQu"
      proposal = "AbstractQueuedLongSynchronizer"
      description = "java.util.concurrent.locks.AbstractQueuedLongSynchronizer"
      textReplacements = ArrayList (
        replace region [0 / length: 0] '
        import java.util.concurrent.locks.AbstractQueuedLongSynchronizer'
      )
      editPositions = ArrayList ()
      kind = "REFERENCE"
    ],
    ContentAssistEntry [
      prefix = "AbstractQu"
      proposal = "AbstractQueuedSynchronizer"
      description = "java.util.concurrent.locks.AbstractQueuedSynchronizer"
      textReplacements = ArrayList (
        replace region [0 / length: 0] '
        import java.util.concurrent.locks.AbstractQueuedSynchronizer'
      )
      editPositions = ArrayList ()
      kind = "REFERENCE"
    ]
  ])
]> but was:<...ntries = ArrayList ([])
]>
```

**Comment by cdietrich** [comment]
org.eclipse.xtext.xbase.ide.contentassist.ClasspathBasedIdeTypesProposalProvider does not seem to work

**Comment by cdietrich** [comment]
fixed in 2.18
